pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -913,7 +913,7 @@ func (tc *TidbCluster) TiCDCLogFile() string {
 		}
 	}
 
-	return "/dev/stderr"
+	return "cdc.log"
```

Comment:
How about setting to `""` as suggested in the issue's description? `We need to set the default value to "".`